yaml/com/ibm/License/Entry/LicenseEntry.interface.yaml

```diff
@@ -0,0 +1,87 @@
+description: >
+    Represents license details of features on the system
+    Customer installs license to unlock certain resources or features
+    Each object implements xyz.openbmc_project.License.Entry interface
+    which has its related properties
+    This object implements xyz.openbmc_project.Object.Delete interface
+    to remove a license support
+properties:
+
+    - name: LicenseName
+      type: string
+      description: >
+         Name of license 
+    - name: State
+      type: enum[self.Status]
+      default: Unknown
+      description: >
+          The current state of the license
+    - name: Id
+      type: string
+      description: >
+         Pretty Name Invenoty.Item
+    - name: SerialNum
+      type: string
+      description: >
+         The serial number of the license
+    - name: Type
+      type: enum[self.Type]
+      default: Trial
+      description: >
+          The current type of the license
+    - name: AuthorizationType
+      type: enum[self.AuthorizationType]
+      default: Unlimited
+      description: >
+          The current Authorization type of the license
+    - name: ExpirationDate
+      type: string
+      description: >
+         The date when the license expires
+    - name: AuthDeviceNumber
```

Comment:
Some of the names like this do not match with redifish names. Is that fine ?

Reply:
Yes. We will map them at bmcweb code to the appropriate property